### PR TITLE
Fix visual regression in `BeamlineSetupContainer`

### PR DESCRIPTION
**Pull request by axelboc**
Introduced in #974 (more specifically in [this file](https://github.com/mxcube/mxcubeweb/pull/974/files#diff-dcda9e2584efaa9d2a804f1986c4b67ec10a81c01af8625119c84a715a0d2de5)), probably when rebasing onto `develop`.

### CODE REVIEW

File: ui/src/containers/BeamlineActionsContainer.jsx [hunk 1]
```diff
@@ -76,124 +72,152 @@ class BeamlineActionsContainer extends React.Component {
     const defaultDialogPosition = { x: -100, y: 100 };
 
     return (
-      <Row>
-        <Col xs={12}>
```
Comment (axelboc) [inline ui/src/containers/BeamlineActionsContainer.jsx:80]: Unrelated: I'm removing an unnecessary row/col wrapping in this component. The rest is Prettier formatting (ignore whitespace changes for a cleaner diff).

File: ui/src/containers/BeamlineSetupContainer.jsx [hunk 2]
```diff
@@ -135,42 +142,50 @@ class BeamlineSetupContainer extends React.Component {
...
+      );
       components.push(
         <td
-        key={`bs-val-${uiprop.label}`}
-        className='pe-3'
-        style={{
-          fontWeight: 'bold',
-          border: '0px',
-          borderRight: uiprop_list.length != uiprop_list.indexOf(uiprop) + 1 ? '1px solid #ddd': '',
-          padding: '0em'}}>
```
Comment (axelboc) [inline ui/src/containers/BeamlineSetupContainer.jsx:152]: Styling regressions here

File: ui/src/containers/BeamlineSetupContainer.jsx [hunk 3]
```diff
@@ -185,49 +200,47 @@ class BeamlineSetupContainer extends React.Component {
...
-      style={{
-        background: '#FAFAFA',
-        borderBottom: '1px solid lightgray',
-        paddingBottom: '0em',
-      }}
-      className="beamline-status ps-3 pe-3"
-      id="bmstatus"
-      bg='light'
-      expand="lg"
-    >
```
Comment (axelboc) [inline ui/src/containers/BeamlineSetupContainer.jsx:205]: ... and here

**Comment by marcus-oscarsson** [comment]
Well done :)

**Pull request merged**